docs/slides/content/social-contracts.md

```diff
@@ -93,12 +93,13 @@ If you will work in multiple teams think about what are the aspects of your Soci
 </br></br>
 
 1. In table groups, work together to:
-    - Establish a team name
-    - Talk about how you want to work together this week
+    - Choose a team name
+    - Discuss how you want to work together this week
     - Write your Social Contract
     - Record your team button
 </br></br>
-2. Think about any aspects of your Social Contract that are appropriate for the entire class so we can write the class Social Contract
+<!-- There's nothing in the slides about this "team button" so it's not clear what it means here.-->
```

Comment:
@springdo one comment here about "record a team button", not sure what that is/means.

Reply:
Other requests/comments resolved. :)

Reply:
hehe good question - let me figure it out 🙈

Reply:
So I found out that this is these physical buttons that we use in our face-to-face engagements. It is known to our facilitators but we can update it as I suggested if it is good, else we can leave it as it is. Thank you.
```suggestion
```